bluepysnap/cli.py

```diff
@@ -28,11 +28,13 @@ def cli(verbose):
         "edge indices are correct, etc"
     ),
 )
-def validate(config_file, skip_slow):
+@click.option("--only-errors", is_flag=True, help=("Only print fatal errors (ignore warnings)"))
```

Comment:
Minor, redundant parentheses 
```suggestion
@click.option("--only-errors", is_flag=True, help="Only print fatal errors (ignore warnings)")
```